libs/@local/hash-backend-utils/src/pg-tables.ts

```diff
@@ -2,74 +2,106 @@
  * This module contains type definitions for tables in the Postgres database. Column
  * names are converted from snake_case to camelCase for consistency.
  */
+import { SupportedRealtimeTable } from "./realtime";
 import { Wal2JsonMsg } from "./wal2json";
 
-export class Entity {
-  constructor(
-    /**
-     * @todo Update table definition when provenance info is added for updates
-     *   https://app.asana.com/0/1202805690238892/1202848989198291/f
-     */
-    public entityId: string,
-    public version: string,
-    public entityTypeVersionId: string,
-    public properties: Record<string, unknown>,
-    public createdBy: string,
-  ) {}
+export type EntityEdition = {
+  entity_edition_id: string;
+  properties: Record<string, unknown>;
+  left_to_right_order: number;
```

Comment:
Shouldn't this be optional? 🤔

Reply:
Where's the left entity ID and right entity ID?

Reply:
Yes they should. These types are only mapping the tables, and the table doesn't contain left and right entity ID. These are stored in `entity_has_left_entity` and `entity_has_right_entity` tables which i'm not joining currently

Reply:
Maybe I don't understand what this service actually does, but doesn't that mean we aren't correctly recreating entities if we're missing a good portion of their data?

Reply:
The full entity isn't recreated, no. We can do that, but that requires doing follow-up joins on the query, resulting in a lot of follow-up reads. 
I think the service should be considered a notification service that happens to provide some payload. I would be a little cautious to spend a lot of time making the change streams reflect every materialized result form the DB for now as it could blow up the scope a lot without needing the full view. And as we've pointed out, we're already duplicating the database tables, duplicating queries/join logic seems like something we shouldn't do (right now anyway)